File: test/export/vb/03-estructuras.vb
Module SIN_TITULO
	Sub Main()
		Dim c As Double
		c = 0
		While c<=10
			c = c+1
			Console.WriteLine(c)
		End While
		c = 0
		Do
			c = c+1
			Console.WriteLine(c)
		Loop Until c=10
		c = 0
		Do
			c = c+1
			Console.WriteLine(c)
		Loop While c<10
		If c=10 Then
			Console.WriteLine("Si")
		End If
		If c=10 Then
			Console.WriteLine("Si")
		Else
			Console.WriteLine("No")
		End If
	End Sub
End Module

File: test/export/cs/03-estructuras.cs
using System;
namespace PSeInt {
	class sin_titulo {
		static void Main(string[] args) {
			double c;
			c = 0;
			while (c<=10) {
				c = c+1;
				Console.WriteLine(c);
			}
			c = 0;
			do {
				c = c+1;
				Console.WriteLine(c);
			} while (c!=10);
			c = 0;
			do {
				c = c+1;
				Console.WriteLine(c);
			} while (c<10);
			if (c==10) {
				Console.WriteLine("Si");
			}
			if (c==10) {
				Console.WriteLine("Si");
			} else {
				Console.WriteLine("No");
			}
		}
	}
}
